src/python/director/screengrabberpanel.py

```diff
@@ -27,9 +27,11 @@ def __init__(self, widgets):
 
 
 class ScreenGrabberPanel(object):
+    # HACK
+    instance = None
 
     def __init__(self, view):
-
+        ScreenGrabberPanel.instance = self
```

Comment:
we should remove this.  I can help show you how to get the instance without adding this variable.  I might propose that LCMLogPlayer fires off callbacks, and we add a new class called something like `LcmLogVideoRecorder` which listens to the callbacks and has a ref to the screen grabber panel to record movies.